packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -15,6 +19,9 @@ export function DiscoverSection({
   enableSearch,
   ...defaultRenderSectionProps
 }: DiscoverSectionProps): ReactElement {
+  const hasCommentFeed = useFeature(feature.commentFeed);
+  const { checkHasCompleted, completeAction } = useActions();
+  const hasCompletedCommentFeed = checkHasCompleted(ActionType.CommentFeed);
```

Comment:
@rebelchris you are missing `isActionsFetched, now the "New" label flashes on each page load since by default `checkHasCompleted` returns `false` until actions are fetched.